docs/quickstart.rst

```diff
@@ -260,7 +260,8 @@ Repository encryption
 ---------------------
 
 Repository encryption can be enabled or disabled at repository creation time
-(the default is enabled, with `repokey` method)::
+(for 1.0.x the default value is `repokey`, for >= 1.1.x setting `--encryption` 
+explicitly is mandatory)::
```

Comment:
good catch! but we need to rephrase that, it's not mandatory, only if you want encryption.

how about:

(for borg 1.0.x, `--encryption repokey` was the default. for borg >= 1.1, you need to give `--encryption MODE` explicitly to enable encryption.)

Reply:
I think this is not correct. borg 1.1.5 says without `-e`:

`borg init: error: the following arguments are required: -e/--encryption`

From [changelog](https://borgbackup.readthedocs.io/en/stable/changes.html#version-1-1-0b4-2017-03-27): "init: the –encryption argument is mandatory now"

Reply:
hmm, right. encryption is not mandatory, but `--encryption` is.